src/pages/reportSighting/ReportForm.jsx

```diff
@@ -214,7 +215,7 @@ export default function ReportForm({
           />
         </>
       )}
-      {sightingType && (
+      {sightingType && !authenticated && (
```

Comment:
@brmscheiner has made the recommendation to me before to handle this logic elsewhere such that we only have one boolean here before the component render.
So, maybe something like 
```
const sightingTypeButNotAuthenticated  = sightingType && !authenticated;
...
{sightingTypeButNotAuthenticated && ( component logic)}
```

Reply:
yep, although it's just for cleanliness so it's a toss up in a situation like this when the variable names are shorter and there's only one expression

Reply:
Applied in https://github.com/WildMeOrg/codex-frontend/pull/275/commits/1bbeec8610cd7b331d896cfcc9525699518c4585. I changed the variable name though, so please make sure it matches the conventions.